FRAMEWORKS.yml

```diff
@@ -231,6 +231,10 @@ go:
     website: www.gorillatoolkit.org/pkg/mux
     version: "1.7"
     language: "1.12"
+  gorouter:
+    website: github.com/vardius/gorouter/wiki
```

Comment:
perhaps better `website: rafallorenz.com/gorouter`

Reply:
I wanted to link to router directly, wiki is always up to date